### PR TITLE
Add um::ole2::OleInitialize, shared::wtypes::DVASPECT

### PR DESCRIPTION
**Pull request by ArturKovacs**
These were missing in order to implement an IDropTarget that handles file drop.

### CODE REVIEW

File: src/shared/wtypes.rs [hunk 1]
```diff
@@ -62,6 +62,12 @@ ENUM!{enum VARENUM {
     VT_RESERVED = 0x8000,
     VT_ILLEGAL = 0xffff,
 }}
+ENUM!{enum tagDVASPECT {
```
Comment (retep998) [inline src/shared/wtypes.rs]: This enum should be named `DVASPECT`.

**Review by retep998 — CHANGES_REQUESTED** [review]

**Comment by ArturKovacs** [comment]
Sorry for the 'use' ordering issues. My attention is all over the place.

Is there a good way I can check these rules locally?

**Comment by retep998** [comment]
`cargo test --all-features` will run all the tests if you want to test locally.

**Review by retep998 — APPROVED** [review]

**Comment by ArturKovacs** [comment]
Thank you next time I'll be wiser and run the tests locally before committing.

Is there anything that prevents this to be merged?

**Comment by ArturKovacs** [comment]
Hello again!
I'm looking forward to seeing this merged.
If there's any reason this can't be merged in this current state I would be happy to follow your instructions.

**Pull request merged**